b2/console_tool.py

```diff
@@ -1238,6 +1245,8 @@ class Sync(Command):
 
     To make the destination exactly match the source, use:
     {NAME} sync --delete --replaceNewer ... ...
+    NOTE: It might not be exact match if the source contains hidden files.
+    If the source file is hidden, the file on the destination won't be deleted even with --delete.
```

Comment:
this is a bug, lets not document it as expected behavior or it will become a part of the interface